Description: Use agent by default
Author: Scott Kitterman <scott@kitterman.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/15485
Forwarded: no
Last-Update: 2012-12-04

Index: b/g10/options.skel
===================================================================
--- a/g10/options.skel
+++ b/g10/options.skel
@@ -199,8 +199,12 @@
 # at ftp.gnupg.org/gcrypt/alpha/aegypten/).  To make use of the agent,
 # you have to run an agent as daemon and use the option
 #
-# use-agent
-# 
+# For Ubuntu we now use-agent by default to support more automatic
+# use of GPG and S/MIME encryption by GUI programs.  Depending on the
+# program, users may still have to manually decide to install gnupg-agent.
+
+use-agent
+ 
 # which tries to use the agent but will fallback to the regular mode
 # if there is a problem connecting to the agent.  The normal way to
 # locate the agent is by looking at the environment variable
